tests/test_geometry.py

```diff
@@ -531,14 +531,31 @@ def test_line_length():
     assert line.length == sqrt(2)
 
 
-def test_arc_get_coordinate_from_percentage_distance():
+def test_arc_get_coordinate_from_fractional_distance():
```

Comment:
Can we make sure this also works for arcs with negative radius

Reply:
Tests added for this